packages/validate/translations/it.js

```diff
@@ -6,6 +6,7 @@ export default {
     MaxLength: 'Inserire un(a) {fieldName} corretto(a) (fino a {params} caratteri).',
     MinMaxLength:
       'Inserire un(a) {fieldName} corretto(a) (tra {params.min} e {params.max} caratteri).',
+    Pattern: 'Kérjük, adjon meg érvényes {fieldName} értéket.',
```

Comment:
I believe you copied here the Hungarian translation, instead of the Italian `IsNumber`

Reply:
trust me I am better than this :D - fixed